src/middleware/router.js

```diff
@@ -4,6 +4,7 @@ import net from 'net'
 import tls from 'tls'
 import logger from 'winston'
 import cookie from 'cookie'
+import { Readable } from 'stream'
```

Comment:
Im not sure this method is being used anywhere and can be removed

Reply:
fixed in https://github.com/jembi/openhim-core-js/pull/1060/commits/80469141b8a989a1a554d41119ce1c757b84cb3d